qemu.go

```diff
@@ -106,6 +108,14 @@ const (
 	maxDevIDSize = 31
 )
 
+const (
+	// NVDIMM device needs memory space 1024MB
+	// See https://github.com/clearcontainers/runtime/issues/380
+	maxMemoryOffset = 1024
+)
+
+const procMemInfo = "/proc/meminfo"
```

Comment:
Please move this to `hypervisor.go` since it can be reused across different hypervisor implementations.